updated code to python3; all tests pass

joshuago78 opened the pull request.

joshuago78 committed:
updated code to python3; all tests pass

kirschbombe commented:
I will test this tomorrow morning, @joshuago78.

kirschbombe commented on README.md:
```diff
@@ -9,23 +9,22 @@ NOID-mint Python Package
+6. Confirm the python version is 3.x by running 
 	$ python --version
 	
 7. Navigate down into the NOID-mint directory
```
7. Navigate down into the SLDP_ARK_mint directory

kirschbombe commented on README.md:
```diff
@@ -9,23 +9,22 @@ NOID-mint Python Package
 	$ python --version
 	
 7. Navigate down into the NOID-mint directory
 	$ cd NOID-mint
```
$ cd SLDP_ARK_mint

kirschbombe commented:
Hi, @joshuago78. Set up and installation went fine, but I ran into an error running the `sinai_work_page_ark.py` script:
```
(arks3) kirschbombe@YRL-ML-DLP003 SLDP_ARK_mint % python sinai_work_page_ark.py 
Traceback (most recent call last):
  File "sinai_work_page_ark.py", line 31, in <module>
    directory = (str(raw_input('File directory:')).strip())+'/'
NameError: name 'raw_input' is not defined
```

joshuago78 committed:
updated script to look for files that start with 'works', not just 'works.csv'

joshuago78 committed:
use Shelfmark as key instead of altidentifier.local

joshuago78 committed:
replaced raw_input() with input()

joshuago78 commented:
OK, I have removed the deprecated `raw_input()` methods. I have also added the other two requirements: using Shelfmark and looking for works*.csv files

kirschbombe commented:
thanks! I will test this in the morning (or tonight if I'm not too tired).

kirschbombe commented:
I tested again today, but received a lot more errors. I put them in a gist here: https://gist.github.com/kirschbombe/fe8050dfeafadbdd2b8976d7d4368788.js

joshuago78 commented:
Closing this PR. Andy and Zoe will take a look at the changes I made and begin a new branch/PR
